<launch>
	<arg name="id"/>
	<node pkg="gitagent" type="agent_run.py" name="brain_node" output="log" launch-prefix="xterm -e">
		<param name="myID" value="$(arg id)" />
		<param name="myTheta" value="0.5" />
		<param name="myDelta" value="0.2" />
		<param name="myDepend" value="10" />
		<param name="provaNr" value="2_1" />
	</node>	<node pkg="gitagent" type="msg_PUnit.py" name="msg_punit" output="log" >
		<param name="myID" value="$(arg id)" />
	</node>
</launch>
